.travis.yml

```diff
@@ -15,12 +15,16 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn
-  - npm run bootstrap
+  - yarn install
+  - yarn run bootstrap
 
 script:
-  - npm run lint
-  - npm run test
+  - node --version
+  - npm --version
+  - yarn --version
```

Comment:
Just to show the versions that it is running on, better to debug if we have an issue with a specific CI.

The same goes for AppVeyor.